RobotCode/src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.commands;
+
+import org.w3c.dom.views.DocumentView;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ArmConstants;
+import frc.robot.subsystems.ArmSubsytem;
+
+public class ArmCommand extends Command
+{
+	public ArmSubsytem armLower;
```

Comment:
We never make two copies of one subsystem, if we had an upper and lower arm we would need a subsystem for each one. I know you removed the upper/lower part in the subsystem but make sure to do it here.